authors/apps/articles/migrations/0002_article_read_time.py

```diff
@@ -1,19 +0,0 @@
-# Generated by Django 2.1.7 on 2019-04-08 15:49
```

Comment:
Going forward, please refrain from tampering with migration files. Always generate a new one rather than making changes to an old one if you have changes in models.py files. This will make it easier to apply migrations without corrupting data if we were working with a database that contains actual data.

Return this file and generate a new file to which you'll add `('profiles', '0001_initial')` at the top of the dependencies list.
Also, undo all changes to `authors/apps/articles/migrations/0001_initial.py`